### PR TITLE
Update zh_cn.lang 

### PR DESCRIPTION
**Pull request by Flithor**
Update Chinese translation file.
(Fixed the wrong change)

### CODE REVIEW

File: src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang [hunk 1]
```diff
@@ -161,13 +163,14 @@ gui.appliedenergistics2.VibrationChamber=谐振仓
 gui.appliedenergistics2.NetworkDetails=网络信息
 gui.appliedenergistics2.StorageCells=ME存储元件
 gui.appliedenergistics2.IOBuses=ME输入/输出总线
-gui.appliedenergistics2.Stores=Stores
+gui.appliedenergistics2.Stores=商店
```
Comment (3TUSK) [inline src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang]: Wrong. “储存” or “存储”

File: src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang [hunk 2]
```diff
@@ -350,7 +353,10 @@ gui.appliedenergistics2.Red=红色
 gui.appliedenergistics2.Black=黑色
 gui.appliedenergistics2.Fluix=Fluix默认色
 
-// waila
+//方块信查看模组配置字段---------------------------------------------------
```
Comment (3TUSK) [inline src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang]: Remove extra comment

File: src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang [hunk 3]
```diff
@@ -304,7 +307,7 @@ gui.tooltips.appliedenergistics2.LevelType=检测类型
 gui.tooltips.appliedenergistics2.LevelType_Energy=能源(ae)
 gui.tooltips.appliedenergistics2.LevelType_Item=物品
 gui.tooltips.appliedenergistics2.InventoryTweaks=InventoryTweak(R键整理)
-gui.tooltips.appliedenergistics2.Mod=Mod
+gui.tooltips.appliedenergistics2.Mod=模组
```
Comment (3TUSK) [inline src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang]: 建议保留 "Mod" 不翻译

File: src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang [hunk 4]
```diff
@@ -359,7 +365,28 @@ waila.appliedenergistics2.Unlocked=未锁定
 waila.appliedenergistics2.Showing=显示
 waila.appliedenergistics2.Contains=容纳
 waila.appliedenergistics2.Channels=%1$d / %2$d 频道
-waila.appliedenergistics2.Crafting=合成中
+waila.appliedenergistics2.P2PUnlinked=未连接
+waila.appliedenergistics2.P2PInputOneOutput=已连接(输入端)
+waila.appliedenergistics2.P2PInputManyOutputs=已连接(输入端)- %d 输出
+waila.appliedenergistics2.P2POutput=已连接(输出端)
+
+// TheOneProbe 一个探测仪
```
Comment (3TUSK) [inline src/main/resources/assets/appliedenergistics2/lang/zh_cn.lang]: No translation

**Comment by fscan** [comment]
Is this ok to merge now?

**Comment by 3TUSK** [comment]
I think so

**Pull request merged**